src/webgpu/api/validation/capability_checks/limits/maxSampledTexturesPerShaderStage.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { range, reorder, kOrderKeys } from '../../../../../common/util/util.js';
+import { kShaderStageCombinationsWithStage } from '../../../../capability_info.js';
+
+import { kLimitBaseParams, makeLimitTestGroup } from './limit_utils.js';
+
+const limit = 'maxSampledTexturesPerShaderStage';
+export const { g, description } = makeLimitTestGroup(limit);
+
+g.test('createBindGroupLayout,at_over')
+  .desc(
+    `
+  Test using at and over ${limit} limit in createBindGroupLayout
+  
+  Note: We also test order to make sure the implementation isn't just looking
+  at just the last entry.
+  `
+  )
+  .params(
+    kLimitBaseParams
+      .combine('visibility', kShaderStageCombinationsWithStage)
```

Comment:
We might want cases like: maxStorageBuffersPerShaderStage vertex bindings and maxStorageBuffersPerShaderStage+1 fragment bindings.

But ISTR we already have tests for that up to the base limit, so maybe that's enough.

Same comment for all of them

Reply:
done